src/WireWriter.cs

```diff
@@ -272,10 +272,10 @@ public void WriteDomainName(string name, bool uncompressed = false)
         /// </remarks>
         public void WriteString(string value)
         {
-            if (value.Any(c => c > 0x7F))
-            {
-                throw new ArgumentException("Only ASCII characters are allowed.");
-            }
+            //if (value.Any(c => c > 0x7F))
+            //{
+            //    throw new ArgumentException("Only ASCII characters are allowed.");
+            //}
```

Comment:
Why are you removing this check. The spec only allows US-ASCII.

Reply:
same as prev